ui/src/components/drafts/form/themes/grommet/templates/ObjectFieldTemplate.js

```diff
@@ -57,9 +56,7 @@ let ObjectFieldTemplate = function(props) {
             title={props.title}
             required={props.schema.required}
             description={
-              props.description ? (
-                <span dangerouslySetInnerHTML={{ __html: props.description }} />
-              ) : null
+              props.description ? <span> {props.description}</span> : null
```

Comment:
Should be left as was - possibilities to display links(`a` tags), MathJax, etc
Investigate if a better way exists (test in statistics questionnaire)